src/llmq/instantsend.cpp

```diff
@@ -538,28 +532,28 @@ bool CInstantSendManager::TrySignInputLocks(const CTransaction& tx, bool fRetroa
 
         // don't even try the actual signing if any input is conflicting
         if (sigman.IsConflicting(params.llmqTypeDIP0024InstantSend, id, tx.GetHash())) {
-            LogPrintf("CInstantSendManager::%s -- txid=%s: sigman.IsConflicting returned true. id=%s\n", __func__,
+            LogPrintf("CInstantSendManager::%s -- txid=%s: sigman.IsConflicting returned true. id=%s\n", nostd::source_location::current().function_name(),
```

Comment:
I personally like `__func__` in this particular case. Not because I like macroses (I hate them), but this macros doesn't seems as a harm.

nostd::source_location::curent().fnction_name() is way too long to type for each logs.
It is fine to use in some util code such as `gsl::pointers` once but that's too many words for too simple things.

Can you drop this sanitizer rule?

I'd prefer to write manually a function name for each instance instead typing nosstd::.... - either std::